doc/_themes/mantella/static/style.css

```diff
@@ -212,7 +212,7 @@ main details pre {
   color: #f8f8f8;
 }
 
-main details > div:nth-child(even) pre {
+main details > div:nth-child(3n) pre {
```

Comment:
Why was this changed (it is save to assume, that I did not know how it looks with these changes)?